demo/app/scripts/controllers/main.js

```diff
@@ -204,6 +204,12 @@ angular.module('angularGanttDemoApp')
                         api.tasks.on.resizeEnd($scope, addEventName('tasks.on.resizeEnd', logTaskEvent));
                     }
 
+                    if (api.tasks.on.drawBegin) {
+                        api.tasks.on.drawBegin($scope, addEventName('tasks.on.drawBegin', logTaskEvent));
+                        //api.tasks.on.draw($scope, addEventName('tasks.on.draw', logTaskEvent));
```

Comment:
Ho yes, just realized it's in the demo sources. Lets keep those comments, people reading sources can see thos events exists, but we don't want to flood the console with logs.